objects/evidences.json

```diff
@@ -0,0 +1,91 @@
+{
+  "caption": "Evidence Artifacts",
+  "description": "A collection of evidence artifacts associated to the activity/activities that triggered a security detection.",
+  "extends": "object",
+  "name": "evidences",
+  "attributes": {
+    "actor": {
+      "description": "Describes details about the user/role/process that was the source of the activity that triggered the detection.",
+      "requirement": "recommended"
+    },
+    "api": {
+      "description": "Describes details about the API call associated to the activity that triggered the detection.",
+      "requirement": "recommended"
+    },
+    "container": {
+      "description": "Describes details about the container associated to the activity that triggered the detection.",
+      "requirement": "recommended"
+    },
+    "connection_info": {
+      "description": "Describes details about the network connection associated to the activity that triggered the detection.",
+      "requirement": "recommended"
+    },
+    "data": {
+      "description": "Additional evidence data that is not accounted for in the specific evidence attributes.<code> Use only when absolutely necessary.</code>",
+      "requirement": "optional"
+    },
+    "database": {
+      "description": "Describes details about the database associated to the activity that triggered the detection.",
+      "requirement": "recommended"
+    },
+    "databucket": {
+      "description": "Describes details about the databucket associated to the activity that triggered the detection.",
+      "requirement": "recommended"
+    },
+    "device": {
```

Comment:
I'm not seeing `device` `email` `user` or `url` in the upstream definition of this object. Are these specific use cases on our end?

Reply:
Did you check the `1.3.0-dev`? schema.ocsf.io defaults to 1.2.0, rather than the bleeding edge.

Reply:
Ahh yup that's the issue. I had changed the version earlier but lost it when I reloaded the page, could call